.github/workflows/benchmark_asterinas.yml

```diff
@@ -94,31 +94,38 @@ jobs:
     - name: Set up benchmark configuration
       run: |
         ALERT_THRESHOLD=$(jq -r '.alert_threshold' test/benchmark/${{ matrix.benchmark }}/config.json)
-        echo "ALERT_THRESHOLD=$ALERT_THRESHOLD" >> $GITHUB_ENV
-        
         ALERT_TOOL=$(jq -r '.alert_tool' test/benchmark/${{ matrix.benchmark }}/config.json)
-        if [ "${ALERT_TOOL}" = "null" ]; then
-          ALERT_TOOL="customSmallerIsBetter"
-        fi
-        echo "ALERT_TOOL=$ALERT_TOOL" >> $GITHUB_ENV
-        
+        TITLE=$(jq -r '.title' test/benchmark/${{ matrix.benchmark }}/config.json)
         DESCRIPTION=$(jq -r '.description' test/benchmark/${{ matrix.benchmark }}/config.json)
-        if [ -z "$DESCRIPTION" ]; then
-          DESCRIPTION=""
-        fi
+        SHOW_IN_OVERVIEW=$(jq -r '.show_in_overview' test/benchmark/${{ matrix.benchmark }}/config.json)
+
+        ALERT_TOOL=${ALERT_TOOL:-"customSmallerIsBetter"}
+        TITLE=${TITLE:-""}
```

Comment:
Could you tell any differences between the outcome of these commands:

```bash
TITLE=$(jq -r '.title' test/benchmark/${{ matrix.benchmark }}/config.json)
TITLE=${TITLE:-""}
echo "TITLE=$TITLE" >> $GITHUB_ENV
```

and that of a more simplified form:

```bash
TITLE=$(jq -r '.title' test/benchmark/${{ matrix.benchmark }}/config.json)
echo "TITLE=$TITLE" >> $GITHUB_ENV
```

?

Reply:
Sorry, I didn't realize it before. There is no difference. I made a mistake. `jq` will set the missing fields to `null` instead of `""`.  `TITLE=${TITLE:-""}` will not work. I fix the code here.

For `TITLE`, I think using the benchmark name as an alternative is a better choice. And for `DESCRIPTION`, set it to empty if it is not set. Now it works like:
```bash
        ALERT_THRESHOLD=$(jq -r '.alert_threshold' test/benchmark/${{ matrix.benchmark }}/config.json)
        ALERT_TOOL=$(jq -r '.alert_tool' test/benchmark/${{ matrix.benchmark }}/config.json)
        TITLE=$(jq -r '.title' test/benchmark/${{ matrix.benchmark }}/config.json)
        DESCRIPTION=$(jq -r '.description' test/benchmark/${{ matrix.benchmark }}/config.json)
        SHOW_IN_OVERVIEW=$(jq -r '.show_in_overview' test/benchmark/${{ matrix.benchmark }}/config.json)

        if [ "$ALERT_THRESHOLD" = "null" ]; then
          ALERT_THRESHOLD="130%"
        fi
        if [ "$ALERT_TOOL" = "null" ]; then
          ALERT_TOOL="customSmallerIsBetter"
        fi
        if [ "$TITLE" = "null" ]; then
          TITLE="${{ matrix.benchmark }}"
        fi
        if [ "$SHOW_IN_OVERVIEW" = "null" ]; then
          SHOW_IN_OVERVIEW="false"
        fi

        echo "ALERT_THRESHOLD=$ALERT_THRESHOLD" >> $GITHUB_ENV
        echo "ALERT_TOOL=$ALERT_TOOL" >> $GITHUB_ENV
        echo "TITLE=$TITLE" >> $GITHUB_ENV
        echo "DESCRIPTION=$DESCRIPTION" >> $GITHUB_ENV
        echo "SHOW_IN_OVERVIEW=$SHOW_IN_OVERVIEW" >> $GITHUB_ENV
```